### DISCUSSION TITLE
Comments should not (and no longer) be indexed.

### DISCUSSION THREAD

**Discussion by CalinZBaenen** [forem/forem#19165] [Feature Requests]
([This is based on the DEV post I made](https://dev.to/baenencalin/suggestion-for-dev-30c5); I emailed [yo](https://dev.to/yo) and they suggested I put the suggestion here.)

## Suggestion
Comments on [DEV](https://dev.to) should no longer be indexed (including already posted ones).  
This would make it so if you looked up something on Google you wouldn't find something irrelevant or in the middle of a conversation.

## Rationale
My reasoning for making this suggestion is because recently I [looked up my name, "Calin Baenen", using Google](https://www.google.com/search?q=calin+baenen) and the second result was [a random comment replying to me](https://dev.to/baenencalin/comment/215k4):
![Image of Google's search results for "Calin Baenen".](https://user-images.githubusercontent.com/50878485/220498658-65dd11a2-f472-47cd-bef1-9855f95df699.png)
![Image of the comment replying to me.](https://user-images.githubusercontent.com/50878485/220498832-f518283f-6174-4562-bb7a-ed12865802b9.png)

In addition, I don't believe it's standard for sites to index comments \[\<\-\- Take that with a grain of salt. I prob. don't know what I'm talking about.\]; the only site I've seen do this is [Twitter](https://twitter.com).

**Comment by michael-tharrington** [comment]
Thanks for bringing this up, Calin!

**Reply by CalinZBaenen** [reply]
Of course.

**Comment by CalinZBaenen** [comment]
As an update, it seems that the content displayed is subject to change, showing what I most recently wrote: ![Updated image of Google's search results for "Calin Baenen".](https://user-images.githubusercontent.com/50878485/222055658-5b5dff36-c2ab-4c31-89d7-45d23730f2e6.png)

**Reply by CalinZBaenen** [reply]
Nevermind, apparently that's just Google's filter feature being generous and turned out by default.  
It could theoretically look like **this**: ![Image of Google's search results not being filtered.](https://user-images.githubusercontent.com/50878485/222057196-6d302450-0129-433c-8e32-996252afb26e.png)

💀

**Reply by CalinZBaenen** [reply]
(CC: @benhalpern)

Another exhibit:
![Image of a lot of DEV related results.](https://user-images.githubusercontent.com/50878485/222057612-ba83a3b7-ae77-47b0-9386-3327167c97fb.png)
